pkg/cluster/prepare.go

```diff
@@ -144,7 +155,10 @@ func (a *Applier) Apply(folder string, kubeContext string, kubeConfig string, dr
 		fmt.Print(string(output))
 		return nil
 	}
-	// In a follow up ticket, make this wait for all the possible resources to be condition=available.
-	fmt.Println("install finished")
+	fmt.Print("Waiting for resources to be verified...")
```

Comment:
```suggestion
	fmt.Print("Waiting for resources to be ready...")
```

consistent with the error message below